salt/modules/win_update.py

```diff
@@ -82,6 +84,11 @@ def __virtual__():
     '''
     Only works on Windows systems
     '''
+    salt.utils.warn_until(
+        'Oxygen',
+        'The \'win_update\' module is deprecated, and will be removed in Salt '
+        '{version}. Please use the \'win_wua\' module instead.'
+    )
```

Comment:
Please move the deprecation into the is_windows check since non windows users might not be interested in seeing that deprecation warning?
I could be wrong though....